ci/scripts/install-linuxbrew.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
@dviramontes We should remove this file.

Reply:
ah lolz, yeah we should!